x/txfees/types/options.go

```diff
@@ -0,0 +1,80 @@
+package types
+
+import (
+	"fmt"
+	servertypes "github.com/cosmos/cosmos-sdk/server/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cast"
+)
+
+// If Options are not set in a config somewhere,
+// use defaults to preserve functionality with old node software
+
+// TODO: Bump after next minor version. (in 6.2+)
+var DefaultMinGasPriceForArbitrageTx = sdk.ZeroDec()
+var DefaultMinGasPriceForHighGasTx = sdk.MustNewDecFromStr("0.01")
```

Comment:
```suggestion
var DefaultMinGasPriceForHighGasTx = sdk.ZeroDec()
```

Reply:
Do you think we should make the default for new people on the software to be 0, or .01?

Reply:
We can start as with arbitrage txes - with default zero, but active community validators can set it to `.01` to prevent spam waves.

Reply:
Ok cool, and everyone who regenerates the app.toml will get the new setting